_data/sites.yml

```diff
@@ -304,9 +304,9 @@
   last_checked: 2022-05-15
 
 - domain: blmayer.dev
-  url: https://blmayer.dev/
-  size: 20.4
-  last_checked: 2022-03-25
+  url: https://www.blmayer.dev/
+  size: 5.5
```

Comment:
Could you provide a GTMetrix link to verify this?

Reply:
https://gtmetrix.com/reports/www.blmayer.dev/boDvNHqJ/

Reply:
According to that, the site is 7.53kb. could you update that?

Reply:
Yes, I updated.